lib/Driver/Compilation.cpp

```diff
@@ -206,8 +206,9 @@ namespace driver {
     /// Jobs that incremental-mode has decided it can skip.
     CommandSet DeferredCommands;
 
-    /// Jobs in the initial set with Condition::Always, or lacking existing
+    /// Jobs in the initial set with Condition::Always, and having an existing
     /// .swiftdeps files.
```

Comment:
This change seems incorrect. NewlyAdded files would also show up here (at least they should).

Reply:
On second thought, maybe not. Maybe the variable name just needs to change.

Reply:
Here is the code from `Compilation.cpp`:
````
       case Job::Condition::Always:
          if (Comp.getIncrementalBuildEnabled() && !DependenciesFile.empty()) {
            // Ensure dependents will get recompiled.
            InitialOutOfDateCommands.push_back(Cmd);
            // Mark this job as cascading.
            DepGraph.markIntransitive(Cmd);
          }
          LLVM_FALLTHROUGH;
````

Reply:
Please look at the code above and see if the code is wrong, or I'm reading it wrong, or what? Am I missing another place where jobs get added?

Reply:
Oops, didn't see your second thought, messages crossed in progress. Happy to change the name if you like. Let me know.

Reply:
Yeah, given that this is the only place this gets set, I suggest "InitialCascadingCommands" or something.

Reply:
OK, will do!